styles/index.css

```diff
@@ -173,3 +184,68 @@ body.dark-mode {
 .theme-toggle-icon {
   @apply ml-1;
 }
+
+/* Modal
+============================== */
+
+.overlay {
```

Comment:
@icegotcha Please use Tailwind utility classes whenever possible. Custom CSS should be used as a last resort. Thanks.

Reply:
Done.